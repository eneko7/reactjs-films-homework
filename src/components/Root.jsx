import { hot } from 'react-hot-loader';
import App from './App/App.jsx';

export default hot(module)(App);