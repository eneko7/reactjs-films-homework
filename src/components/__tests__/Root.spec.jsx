import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Root from '../Root';

const renderer = new ShallowRenderer();
renderer.render(<Root />);
describe('Root', () => {
  it('renders correctly', () => {
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
