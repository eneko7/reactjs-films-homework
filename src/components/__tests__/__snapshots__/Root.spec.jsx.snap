// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Root renders correctly 1`] = `
<div
  className="main_container"
>
<<<<<<< HEAD
  <header
    className="app_header"
  >
    <div
      className="header_top"
    >
      <span
        className="header_top_span"
      >
        FILMS
      </span>
      <div
        className="header_top_search"
      >
        <input
          className="header_top_search_input"
          id="search_input"
          placeholder="Searh..."
          type="text"
        />
        <button
          className="header_top_search_label"
          type="button"
        />
      </div>
    </div>
    <div
      className="header_bottom"
    >
      <div
        className="active header_bottom_left"
      >
        <div
          className="full_description"
        >
          <span
            className="full_description_filmName"
          >
            THE JUNGLE BOOK
          </span>
          <div
            className="full_description_filmDescr"
          >
            <span
              className="full_description_filmDescr_item"
            >
              Adventure
            </span>
            <span
              className="full_description_filmDescr_item"
            >
               Drama
            </span>
            <span
              className="full_description_filmDescr_item"
            >
               Family
            </span>
            <span
              className="full_description_filmDescr_item"
            >
               Fantasy
            </span>
            <span
              className="full_description_filmDescr_item"
            >
               |
            </span>
            <span
              className="full_description_filmDescr_item"
            >
               1h 46m
            </span>
          </div>
        </div>
        <div
          className="bottom_left_filmRate"
        >
          <div
            className="bottom_left_filmRate_stars_outer"
          >
            <div
              className="bottom_left_filmRate_stars_outer_stars_inner"
              id="rate"
              style={
                Object {
                  "width": "94%",
                }
              }
            />
          </div>
          <div
            className="bottom_left_filmRate_number"
          >
            <span>
              4.8
            </span>
          </div>
        </div>
      </div>
      <div
        className="bottom_right"
      >
        <div
          className=" bottom_right_text_descr"
        >
          <p
            className="bottom_right_text_descr_p"
          >
            There are growing dangers in the wizarding world og 1926 New York. Something mysterious is leaving a path of destruction in the streets, threatening to expose the wizarding
          </p>
        </div>
        <div
          className=" bottom_right_buttons_box"
        >
          <button
            className="bottom_right_buttons_box_button_watch bottom_right_buttons_box_button"
            type="button"
          >
            Watch Now
          </button>
          <button
            className="bottom_right_buttons_box_button_view bottom_right_buttons_box_button"
            onClick={[Function]}
            type="button"
          >
            View Info
          </button>
        </div>
      </div>
    </div>
  </header>
=======
>>>>>>> 8084142df0ee52241f61214375ea1ce5322e752b
  <footer
    className="footer"
  >
    <span
      className="footer_films"
    >
      FILMS
    </span>
    <span
      className="footer_copyright"
    >
      Copyright Â©
      2018
      FILMS.
      <span
        className="copyright_signature"
      >
        IHAR KARPUK
      </span>
    </span>
  </footer>
</div>
`;
